test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -10137,7 +10138,7 @@ BOOST_AUTO_TEST_CASE(copy_internal_function_array_to_storage)
 			function() internal returns (uint)[20] x;
 			int mutex;
 			function one() returns (uint) {
-				function() internal returns (uint)[20] xmem;
+				function() internal returns (uint)[20] memory xmem;
```

Comment:
Maybe I could have noticed that a variable called ``xmem`` was not actually supposed to be in storage :smile:.